tests/manage/pv_services/test_del_mon_service_and_create_pvc.py

```diff
@@ -0,0 +1,212 @@
+import json
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    skipif_external_mode,
+    E2ETest,
+    tier4a,
+    ignore_leftovers,
+)
+from ocs_ci.helpers.helpers import modify_deployment_replica_count
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP, get_services_by_label
+from ocs_ci.ocs.resources.pod import get_mon_pods, get_operator_pods, run_io_in_bg
+from ocs_ci.utility.utils import ceph_health_check
+
+log = logging.getLogger(__name__)
+
+POD_OBJ = OCP(kind=constants.POD, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE)
+
+
```

Comment:
shouldn't we mention the bz # that is automated by this test?

Reply:
added